editions/tw5.com/tiddlers/messages/SampleWindowTemplate.tid

```diff
@@ -0,0 +1,10 @@
+created: 20211109165213041
+modified: 20211109175739768
+tags: 
+title: SampleWindowTemplate
+
+
+| ''Name:'' |<$edit field=name tag="input" />|
```

Comment:
There is a little problem with the spaces in the `| ''Name:'' |` imo it should be `|''Name:'' |` Same as the Rank ... 
The tags: field can be removed, if there are no tags